services/src/main/java/org/keycloak/services/util/CookieHelper.java

```diff
@@ -96,7 +98,7 @@ public static Set<String> getCookieValue(KeycloakSession session, String name) {
         Set<String> ret = getInternalCookieValue(session, name);
         if (ret.size() == 0) {
             String legacy = name + LEGACY_COOKIE;
-            logger.debugv("Could not find any cookies with name '{0}', trying '{1}'", name, legacy);
+            logger.debugv(MessageFormat.format("Could not find any cookies with name {0}, trying {1}", name, legacy));
```

Comment:
The `debugv()` method already applies the `MessageFormat` internally, so there is no need to call it explicitly.

```suggestion
            logger.debugv("Could not find any cookies with name {0}, trying {1}", name, legacy);
```